web/source/dom/domEventHandlers.ts

```diff
@@ -791,10 +799,8 @@ namespace com.keyman.dom {
       }
       
       // Identify the input element from the touch event target (touched element may be contained by input)
-      if(target.className == null || target.className.indexOf('keymanweb-input') < 0) target=<HTMLElement> target.parentNode;
-      if(target.className == null || target.className.indexOf('keymanweb-input') < 0) target=<HTMLElement> target.parentNode;
-      if(target.className == null || target.className.indexOf('keymanweb-input') < 0) return;
-      
+      target = findTouchAliasTarget(target);
```

Comment:
I guess technically `findTouchAliasTarget` can return `null` (even if that is circumvented with a throw for logging purposes at present) so we should probably handle that case with a bare `return` as in the old code... Missed that last time!